lib/compiler.js

```diff
@@ -509,3 +545,19 @@ function resolveRelativePaths(relativePaths, appRootDir) {
     }
   });
 }
+
+function normalizeMixinName(str, options) {
+  var normalization = options.normalization;
+  if (normalization === false || normalization === 'none') return str;
+  if (normalization === 'dasherize') {
+    str = String(str).replace(/[\W_]/g, ' ').toLowerCase();
+    str = str.replace(/\s+/g, '-');
+  } else if (typeof normalization === 'function') {
+    str = normalization(str);
+  } else { // classify
+    str = String(str).replace(/[\W_]/g, ' ').toLowerCase();
+    str = str.replace(/(?:^|\s|-)\S/g, function(c) { return c.toUpperCase(); });
+    str = str.replace(/\s+/g, '');
```

Comment:
Please detect the case when `normalization` is defined and it's not any of the values handled above. You should either throw an error or at least report a warning. Otherwise it's very difficult to troubleshoot the situation when you e.g. make a typo in the normalization value and I don't want to get github issues reported by users in this situation.